package.json

```diff
@@ -9,7 +9,8 @@
     "doclint": "npm run doc -- -t templates/silent",
     "test": "npm run lint && npm run build && npm run doclint",
     "build": "webpack -p",
```

Comment:
@tbroyer I found out this `build` is not executed on `npm pack`, hence the `prepublish` below.